### DISCUSSION TITLE
MRTK3 with Pico Neo 3 UltraLeap

### DISCUSSION THREAD

**Discussion by stgrosshh** [microsoft/MixedRealityToolkit-Unity#11397] [Q&A]
Hi there,
has anybody a working example of MRTK3 for Pico Neo3 with UltraLeap hand tracking?
The standard setup (just a MRTK XR Rig Prefab) and basic OpenXR setup with UltraLeap OpenXR and Pico OpenXR seem not work.
OpenXR Ultra Leap Examples which contain special components are working though.
Maybe some additional Ultra Leap specific components must be in the scene? But this would not match the "no special config required" approach of MRTK.
Obviously I make some mistake here.

Any help appreciated.
Cheers
 Stefan

**Comment by Ultraleap-Dan** [comment]
Hi, we would expect this to work as Pico are not using the standard loader yet which means that our OpenXR API Layer won't be active. We are looking at a workaround whereby we include the Ultraleap API Layer in the built application itself rather than installing it separately. 

If you do have any contacts at Pico you could also request that they switch to the standard loader as this would be a permanent fix.
